### ISSUE TITLE
text type for lat/lng minutes and seconds

### ISSUE THREAD

**Issue by brianaAtWork** [FihoFi/ntm_common_xml#4]
Please PLEASE please....can we **NOT** make the _minutes_ and _seconds_ a **STRING**!???? That will imply format information because really, just like lat/lng _degree_ it should be int.

As a programmer, I want to take those values and do direct calculations on them WITHOUT having to process them and convert them. The symbols should be put in after the fact with the generating scripts for portrayal (or transforms) to ensure system, language, and encoding compatibility.

**Comment by FihoFi** [comment]
The variables **latitudeDecimal** and **longitudeDecimal** (of type **float**) should be used for direct calculations of the positions. For this use you should not need the sexagesimal presentation  variables degrees, minutes and seconds. The sexagsimal representation allows for displaying combinations like dd.mm dd mm ss, dd mm.mm, ddmm.mmm exactly similarly to the way encoded (and displayed on the chart to be corrected).

**Comment by brianaAtWork** [comment]
That's fine...but with that logic degree should then be text...and with that logic get rid of three variables and merge them all together for a final formatted string....however, I think we should simplify the data structure and not allow the text version so everyone could format it on their own depending on their preference. Ideally, this file should separate data from presentation.

_(sexagesimal? Freudian typo?)_

**Comment by brianaAtWork** [comment]
> _(sexagesimal? Freudian typo?)_
it was actually a joke ;-)

**Comment by FihoFi** [comment]
Agree, that degree could also be converted into a text- format. There is a specific need to preserve the original accuracy and format of the position. It was agreed (after lengthy discussions at the workshop) to separate the parts into separate variables.

**Comment by brianaAtWork** [comment]
> There is a specific need to preserve the original accuracy and format of the position
understood.

>It was agreed (after lengthy discussions at the workshop) to separate the parts into separate variables.
I didn't see anything in the workshop notes about this. Would you explain why? It seems that if "There is a specific need to preserve the original accuracy and format of the position" then it seems logical to keep it all together EXACTLY as it was received. (reducing structure from three to one variable)

**Comment by FihoFi** [comment]
I am not able to give you the exact reasons, but I know several options were discussed (including the option with one formatted string). This chosen option is probably the option that was deemed as the best fit considering the scope, current use and existing processes that will be used to send and receive the data.
